RHEL6_7/system/initscripts/ifcfg/check

```diff
@@ -124,18 +133,23 @@ I case 3, to be completely correct you could check that user is not using rhel7
                 if addr:
                     ethx_with_addr_count +=1
                 else:
+                    ethx_without_addr_count +=1
                     log_slight_risk(full_path + " is old style ethX name without HWADDR, its name can change after the upgrade.")
```

Comment:
its name CAN change or WILL change?

Reply:
yes CAN is correct (explained above)

Reply:
I'd use MAY, but I'm a bit late to the party :)

Reply:
I'd use MIGHT, but I didn't:(